Mailer/README.md

```diff
@@ -520,6 +520,112 @@ Response:
 
 ---
 
+#### POST `/api/v1/users/is-user-unsubscribed`
+
+API call that checks if user unsubscribed given newsletter.
+
+##### *Headers:*
+
+| Name | Value | Required | Description |
+| --- |---| --- | --- |
+| Authorization | Bearer *String* | yes | API token. |
+
+##### *Body:*
+
+```json5
+{
+  "user_id": 1, // Integer; ID of user
+  "list_id": 1 // Integer; ID of newsletter
+}
+```
+
+##### *Example:*
+
+```shell
+curl -X POST \
+  http://mailer.remp.press/api/v1/users/is-user-unsubscribed \
+  -H 'Authorization: Bearer XXX' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"user_id": 123,
+	"list_id": 1
+}'
+```
+
+Response:
+
+```json5
+true
+```
+
+---
+
+#### POST `/api/v1/users/user-preferences`
+
+API call to get subscribed newsletters and their variants.
+
+##### *Headers:*
+
+| Name | Value | Required | Description |
+| --- |---| --- | --- |
+| Authorization | Bearer *String* | yes | API token. |
+
+##### *Body:*
+
+```json5
+{
+  "user_id": 1, // Integer; ID of user
+  "email": "test@test.com", // String; Email from which was the newsletter subscribed
```

Comment:
The statement in the comment is not correct - if email changes, client should be asking for preferences with the new email, not the one used from which newsletter was subscribed. Please change to:

> email to get preferences for